package.json

```diff
@@ -27,5 +27,8 @@
     "workspaces": [
         "packages/backend",
         "packages/frontend"
-    ]
+    ],
+    "dependencies": {
+        "@corvu/tooltip": "^0.2.1"
```

Comment:
The dependency should go in `packages/frontend/package.json` rather than the top-level `package.json`.

Reply:
Thanks! That should be fixed now :)